en/SharedBibFile.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sharing a BibLaTeX Database
+---
+
+# Sharing a Bib(La)TeX Database
+
+JabRef allows to share both Bib(La)TeX database and [SQL database](SQLDatabase).
+
+When sharing a Bib(La)TeX database, JabRef automatically recognizes a change in the `bib` file on disk and notifies the user of it.
+This works well on network drives.
+
+*Note:* the use of a version control system (SVN, git, etc.) is recommended as this will allow for reverting changes.
+
+To make the sharing of a Bib(La)TeX database easier, it is recommanded to set specific database properties.
+In the menu **File -> database properties**:
+- Select `UTF-8` as encoding.
+- Define a `General file directory`, which will be used to store shared PDF (and other) files.
+- Define a sort order (`year`, `author`, `title` is recommended).
```

Comment:
Is this really helping? I would expect "original order" to produce much less conflicts, but I have no actual experience.